src/SmartComponents/ActivityTable/Columns.js

```diff
@@ -15,10 +16,34 @@ TaskNameCell.propTypes = {
   index: propTypes.number,
 };
 
+const StatusCell = ({ status }) => {
+  if (status === 'Completed') {
```

Comment:
How about using a switch statement?

Reply:
You could also go fancy pants and avoid if/else and switch:

```js
const StatusCell = ({ status }) => ({
    Completed: (
        <span style={{ color: '#3E8635', display: 'flex', alignItems: 'center' }}>
            <CheckCircleIcon color="#3E8635" style={{ marginRight: '4px' }} />
            {status}
        </span>
    ),
    Running: (
        <span style={{ color: '#2B9AF3', display: 'flex', alignItems: 'center' }}>
            <InProgressIcon color="#2B9AF3" style={{ marginRight: '4px' }} />
            {status}
        </span>
    )
}[status] || status);
```

However, i think the if/else here is fine, would there be more a mapped object like ^ would be the neatest solution.